trustless-work-cctp-e2e.sh

```
@@ -0,0 +1,545 @@
+
+# Addresses
+ALICE="GCPZCXSEWARYFZAJQEAJORUHZNGJNMQCDYYCDTYTUDD65H4TKKDF65HS"
+APPROVER="GB5RJ5UI7264E5F2S5FA5ZETOTOS4HRWPGAHMHQDAPK6OJRKCXOH2MTI"
+PROVIDER="GCD2CN27C4VUOGF7JNIZYT45MVIIGEDDHYMVCQIGGECAJQFIEDVEYJMY"
+RELEASE_SIGNER="GAIAMLVKFC7NVQXH6UAA7WJCLJY6QGJSU7XBIQMXXGWYJHAWX2K2ZUVB"
+RESOLVER="GCDOXZ3PWII2H6Y23QWAORCJRL7UTDKFBZB7FYGEGP3T6AKB2RB2TWO4"
+PLATFORM="GB3Q632HYA62YVPGUZCOR3YJ4K6Y6HKJBA6DPHT67LF6ZWEPYA5ZSRDM"
+RECEIVER="GB2TGNQ6KXPA3MEXNLGPTRG2UUKG5QWNW6XA2D2JE3NPV6CYD56VXIEW"
+USDC_ISSUER="GBBD47IF6LWK7P7MDEVSCWR7DPUWV3NY3DTQEVFL4NAT4AQH3ZLLFLA5"
+USDC_TRUSTLINE="CBIELTK6YBZJU5UP2WWQEUCYKLPU6AUNZ2BQ4WWFEIE3USCIHMXQDAMA"
+EVM_RECIPIENT="0x18C385B1748ae257179483564A5FB11bc5C18Ab7"
```

Shellcheck (0.11.0)

[warning] EVM_RECIPIENT appears unused. Verify use (or export if used externally).

(SC2034)